hack/make-rules/BUILD

```diff
@@ -41,14 +41,6 @@ sh_binary(
     ],
 )
 
-sh_binary(
```

Comment:
This should be moved to the BUILD file at the new location for this script and any references updated.

Reply:
I moved this file to `test/cmd/kubeadm.sh`, and is now called by `test-cmd` which remains as part of this file.  I didn't see the other scripts in that file part of this BUILD script, but don't fully understand how Bazel builds/uses this file.

Reply:
see the comment at the top of this file, we're _ok_ not to have it as we don't run it this way currently, but it validates what the inputs / dependencies are.

@ixdy thoughts on this? seems like actually maybe we might not need these...

Reply:
I'm pretty sure `test/cmd/BUILD` picks up the renamed file with this:
```
sh_library(
    name = "legacy-script",
    srcs = glob(["*.sh"]),
    data = ["//pkg/kubectl/validation:testdata/v1/validPod.yaml"],
    visibility = ["//visibility:public"],
)
```

And the new script no longer references `hack/ib`

Reply:
thanks, resolving